wordpress/geo-acf/README.md

```diff
@@ -0,0 +1,25 @@
+# Geo ACF
```

Comment:
Do we want to use [WordPress readme formatting](https://johnblackbourn.com/readme-md-plugin-on-wordpress-org/) for this?